examples/tutorials/Protein_Generation.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>Missing some of the standard headers (Colab etc). Can you double check against other tutorials?</p><p> </p><p>It would be good to add on a general overview of the strategy of this tutorial. (I.e Bert doesn't have decoder, we use VAE for that. Give the high level overview of what is being done and why)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4423/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Added colab.</p><p> </p><p>Added a general overview of our approach.</p>